jp_gene_viz/dExpression.py

```diff
@@ -43,7 +44,7 @@ def __init__(self, *args, **kwargs):
         )
         tdd.on_trait_change(self.draw_click, "value")
         tdd.layout.width = "100px"
-        self.title_html = widgets.HTML("Expression Heat Map")
+        self.title_html = widgets.HTML(kwargs["title"])
```

Comment:
This will cause a keyerror if kwargs has no title.  Please change to
```
kwargs.get("title", "Expression Heat Map")
```
to provide a default value.

Also you can change the title after the initialization by setting
```
heatmap.title_html.value = "my heatmap title"
```